docs/modules/merqury.md

```diff
@@ -0,0 +1,9 @@
+---
+Name: Merqury
+URL: https://github.com/marbl/merqury
+Description: >
+  Evaluate genome assemblies with k-mers and more.
+---
+
+The Merqury module parses results generated by
+[Merqury](https://github.com/marbl/merqury).
```

Comment:
Could you please add a little description about what each of these tools does? It's fine to copy and paste text from the tool webpages. Just to give a little more context in the docs for what the tools are doing and why people might be running them.

Reply:
done